lib/js/components/DatePickers/DatePicker/DatePicker.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+	<div
+		class="ds-datePicker"
+		:class="{ '-ds-box': triggerType === DATE_PICKER_TRIGGER_TYPES.BOX }"
+	>
+		<template v-if="triggerType === DATE_PICKER_TRIGGER_TYPES.TILE">
+			<span
+				class="ds-datePicker__label"
+				:class="{
+					'-ds-isUppercase': isLabelUppercase,
+					'-ds-disabled': state === DATE_PICKER_STATES.DISABLED,
+				}"
+				>{{ label }}</span
+			>
+			<ds-tile
+				:text="text"
+				:interactive="isInteractive"
+				:color="color"
+				:state="state"
+				:icon-right="icon"
+				:is-icon-right-hidden-on-mobile="isIconHiddenOnMobile"
+				:eyebrow-text="eyebrowText"
+				@click="toggle"
+			/>
+		</template>
+		<date-picker-box
+			v-else
+			:is-interactive="isInteractive"
+			:placeholder="placeholder"
+			:date="date"
+			:icon="icon"
+			:is-icon-hidden-on-mobile="isIconHiddenOnMobile"
+			:state="state"
+			:color="color"
+			:eyebrow-text="eyebrowText"
+			:is-open="isOpen"
+			@click="toggle"
+		/>
+
+		<span class="ds-datePicker__errorMessage"> {{ errorMessage }} </span>
```

Comment:
```suggestion
		<span v-if="errorMessage" class="ds-datePicker__errorMessage"> {{ errorMessage }} </span>
```
because without message span is still in a DOM 
![Zrzut ekranu 2024-06-28 o 09 50 24](https://github.com/bethinkpl/design-system/assets/4587534/a298b78d-53bb-4df1-abd0-7e6e78983fbe)

Reply:
If we don't place it the component may jump when error occurs

Reply:
but I guess that's correct?

Reply:
I am not really sure

Reply:
let's ask ux

Reply:
https://bethinkteam.slack.com/archives/CPT1HNM50/p1720433344642539?thread_ts=1720431806.808119&cid=CPT1HNM50